src/sonic-yang-models/yang-models/sonic-fabric-monitor.yang

```diff
@@ -32,17 +32,35 @@ module sonic-fabric-monitor{
                 }
 
                 leaf monPollThreshIsolation {
-                    type uint32;
+                    type uint8 {
+                        range 1..10;
+                    }
                     default 1;
                     description "Consecutive polls with higher error rate for isolation.";
                 }
 
                 leaf monPollThreshRecovery {
-                    type uint32;
+                    type uint8 {
+                        range 1..10;
+                    }
                     default 8;
                     description "Consecutive polls with lesser error rate for inclusion.";
                 }
 
+                leaf monCapacityThreshWarn {
+                    type uint8 {
+                        range 5..250;
+                    }
+                    default 10;
+                    description "Percentage of up fabric links.";
```

Comment:
Why is this percentage greater than 100?

Reply:
> Why is this percentage greater than 100?

It was from comparing with other needs, but I forgot we changed design ,  fixed it

Reply:
Will you change the range or description?